packages/podge/podge.0.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: [ "Edgar Aroutiounian <edgar.factorial@gmail.com>" ]
+license: "GPL-3.0"
```

Comment:
Thanks. I'm not ideological, what is the best license for an OCaml library? BSD? Also, when I change the library, say release new version, does that mean I need to do a new pull request?

(I tried opam publish initially but it gave me github permissions issues, i think its a bug in the software)